android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -116,6 +121,20 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             if (rnOptions.hasKey("sendDefaultPii")) {
                 options.setSendDefaultPii(rnOptions.getBoolean("sendDefaultPii"));
             }
+            if (rnOptions.hasKey("enableAutoPerformanceTracking") && rnOptions.getBoolean("enableAutoPerformanceTracking")) {
+                if (RNSentryModule.frameMetricsAggregator == null) {
+                    RNSentryModule.frameMetricsAggregator = new FrameMetricsAggregator();
+                }
+
+                // Only add the current activity to frames metrics tracking if auto performance is on.
```

Comment:
nit: This comment doesn't add any new information, this should be obvious from the boolean check just 5,6 lines above.